assets/css/views/tags.css

```diff
@@ -56,9 +56,12 @@
   font-weight: bold;
   margin-bottom: 5px;
   margin-right: 5px;
-  background: var(--tag-normal-background);
-  border-color: var(--tag-normal-border);
-  color: var(--tag-normal-color);
+  --tag-background: var(--tag-normal-background);
```

Comment:
Looks ugly, I'd move var defs to before any css properties, and maybe separate them from the properties by a space, if stylelint permits that.